### DISCUSSION TITLE
meteor create namespace/repository

### DISCUSSION THREAD

**Discussion by jankapunkt** [meteor/meteor#12437] [Feature Requests]
I was wondering if it would be possible to create custom skeletons, that could be hosted on GitHub and created with `meteor create namespace/repository`, where `namespace` is the name of the user or org and `repository` the name of the repo.

By doing so we could provide community-based sekeletons, that contain additional config like environment-specific test-setup, predefined routes etc.

**Comment by Grubba27** [comment]
I brought up this to the team a while ago, but there are some blank spaces regarding it;  From every standpoint, it would be better, if we could set up a dependabot in the repo, and we could make even more prepared apps.

One main issue for me would be this meteor create command would be something like a git clone for a repo. Let's take an example of this [repo](https://github.com/Grubba27/prisma-template) I have. If I run ```meteor create Grubba27/prisma-template``` it would clone the repo and change something using the conventions from the scaffolds API, or would it just clone it?
